packages/account/src/hooks/useKycAuthStatus.ts

```diff
@@ -7,15 +8,21 @@ type TKycAuthStatusPayload = TSocketRequestPayload<'kyc_auth_status'>['payload']
 /** Custom hook that returns Proof of Identity (POI) and Proof of Address (POA) authentication status details. */
 export const useKycAuthStatus = (payload?: TKycAuthStatusPayload) => {
     const { client } = useStore();
+    const invalidate = useInvalidateQuery();
 
     const { is_authorize } = client;
     const { data, ...kyc_auth_status_rest } = useQuery('kyc_auth_status', {
         payload,
         options: { enabled: is_authorize },
     });
+
+    /** Invalidate KYC auth status API call */
+    const reFetchKycAuthStatus = useCallback(() => invalidate('kyc_auth_status'), [invalidate]);
+
     return {
         /** The KYC auth status */
         kyc_auth_status: data?.kyc_auth_status,
+        reFetchKycAuthStatus,
```

Comment:
```suggestion
        reFetchKycAuthStatus: refetch,
```

Reply:
how do you think, @likhith-deriv ?

Reply:
and also could you please extend tests for the hook after adding more functionality to it 🙏